### DISCUSSION TITLE
OSM map background

### DISCUSSION THREAD

**Discussion by ASamSam** [eclipse-sirius/sirius-desktop#410] [Q&A]
Hello,
i`m pretty sure, on one of your presentations I seen OSM map as a background for the view, but it was just 1 shot, I don`t know details.
Can you please tell me if it is possible to establish a view with nodes and edges on the OSM map? with lat/lng positions?
Sincerely yours, Andrew

**Comment by pcdavid** [comment]
I don't know which presentation you refer to. I seem to remember we showed something like that but from what I remember it was part of an experiment and used Sirius Web, not Sirius Desktop.

Sirius Destkop itself does not support this, and in general assumes the precise positions of the nodes on a diagram have no semantic meaning (i.e. it is not possible to decide where exactly a node will be placed from associated semantic data, and in the other way simply moving or resizing a node will not trigger any tool where you, as the specifier, could react to adjust the underlying semantic data).

**Reply by ASamSam** [reply]
Hello. Thank you for the answer, yes, seems it was about Sirius Web
